src/__tests__/__snapshots__/index.spec.tsx.snap

```diff
@@ -1,3487 +1,37 @@
-        </div>
-        <div style=\\"height: 1px\\"><div style=\\"height: 1px; background-color: lightgray\\"></div></div>
-      </div>
-    </div>
-  </div>
-</div>
-"
-`;
+exports[`HTML Output should match arrays/with-ordered-items.json 1`] = `"array{2}numberstring"`;
 
-exports[`HTML Output given complex type that includes array and complex array subtype, should not ignore subtype 1`] = `
-"<div>
-  <div>
-    <div>
-      <div style=\\"margin-left: 0px\\">
-        <div>
-          <div>
-            <span>object</span>
-            <div>{1}</div>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
-  <div>
-    <div>
-      <div style=\\"margin-left: 20px\\">
-        <div>
-          <div>
-            <div>items</div>
-            <div>
-              <span>null</span>
-              <span>or</span>
-              <span>array[string,number]</span>
-            </div>
-          </div>
-        </div>
-        <div>
-          <div
-            title=\\"This description can be long and should truncate once it reaches the end of the row. If it's not truncating then theres and issue that needs to be fixed. Help!\\"
-          >
-            This description can be long and should truncate once it reaches the end of the row. If it's not truncating
-            then theres and issue that needs to be fixed. Help!
-          </div>
-        </div>
-        <div style=\\"height: 1px\\"><div style=\\"height: 1px; background-color: lightgray\\"></div></div>
-      </div>
-    </div>
-  </div>
-</div>
-"
-`;
+exports[`HTML Output should match arrays/with-single-arrayish-items.json 1`] = `"array[object]{3}codenumberrequiredmsgstringrequiredrefstring"`;
 
-exports[`HTML Output given multiple object and string type, should process properties 1`] = `
-"<div>
-  <div>
-    <div>
-      <div style=\\"margin-left: 0px\\">
-        <div>
-          <div>
-            <div>
-              <span>string</span>
-              <span>or</span>
-              <span>object</span>
-            </div>
-            <div>{1}</div>
-          </div>
-        </div>
-      </div>
-    </div>
-  </div>
-  <div>
-    <div>
-      <div style=\\"margin-left: 20px\\">
-        <div>
-          <div>
-            <div>ids</div>
-            <span>array[integer]</span>
-          </div>
-        </div>
-        <div style=\\"height: 1px\\"><div style=\\"height: 1px; background-color: lightgray\\"></div></div>
-      </div>
-    </div>
-  </div>
-</div>
-"
-`;
+exports[`HTML Output should match combiners/allOfs/base.json 1`] = `"object{5}AllOfMergeObjectsobject{2}Object1Propertystring>= 1 charactersObject2Propertynumber<= 2AllOfMergeValidationsstring>= 1 characters<= 2 charactersAllOfMergeTakeMoreLogicalValidationnumber<= 1AllOfMergeObjectPropertyValidationsobject{1}Propertystring>= 1 characters<= 2 charactersAllOfMergeRefsobject{4}street_addressstringrequiredcitystringrequiredstatestringrequiredzipCodestring"`;
```

Comment:
As mentioned earlier, this not validate nesting correctly.
You'll certainly want to assert the validation, colors (if we still use different colors for types, don't recall), etc. are all in all in the right state.

Reply:
As for the colors we abandoned them for types in new design
https://www.figma.com/proto/IkNc59AU2JN5Kc1qbd2AiB/Elements?node-id=681%3A242&viewport=302%2C436%2C0.980905294418335&scaling=min-zoom